app/datatables/effective_taggings_datatable.rb

```diff
@@ -0,0 +1,16 @@
+class EffectiveTaggingsDatatable < Effective::Datatable
+
+  datatable do
+    col :id, visible: false
+
+    col :tag_name, label: 'Tag'
+    col :tagged_on
```

Comment:
Instead of using `tagged_on` could we just use the `taggable` object.  The datatable should see that it's a resource column and put in links to the show or edit screen if user has permission to edit or show them

Reply:
The columns here should be `col :tag` and `col :taggable`